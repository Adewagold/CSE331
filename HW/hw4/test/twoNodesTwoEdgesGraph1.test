# Tests adding two nodes, one reflexive edge 
# and one edge between two nodes to graph.

# Create a graph
CreateGraph two_nodes_two_edges_graph1

# Add two nodes
AddNode two_nodes_two_edges_graph1 a
AddNode two_nodes_two_edges_graph1 b

# Add two edges (including one reflexive edge)
AddEdge two_nodes_two_edges_graph1 a a AA
AddEdge two_nodes_two_edges_graph1 a b AB

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_two_edges_graph1
ListChildren two_nodes_two_edges_graph1 a
ListChildren two_nodes_two_edges_graph1 b