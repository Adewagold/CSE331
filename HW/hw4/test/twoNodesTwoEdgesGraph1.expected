# Tests adding two nodes, one reflexive edge 
# and one edge between two nodes to graph.

# Create a graph
created graph two_nodes_two_edges_graph1

# Add two nodes
added node a to two_nodes_two_edges_graph1
added node b to two_nodes_two_edges_graph1

# Add two edges (including one reflexive edge)
added edge AA from a to a in two_nodes_two_edges_graph1
added edge AB from a to b in two_nodes_two_edges_graph1

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_two_edges_graph1 contains: a b
the children of a in two_nodes_two_edges_graph1 are: a(AA) b(AB)
the children of b in two_nodes_two_edges_graph1 are: